src/odemis/driver/test/hamamatsurx_test.py

```diff
@@ -418,6 +418,21 @@ def test_TriggerDelay(self):
         with self.assertRaises(IndexError):
             self.delaybox.triggerDelay.value = -1
 
+    def test_Shutter(self):
+        """Test Shutter state VA of streak unit."""
+        # set shutter VA
+        self.streakunit.shutter.value = False
+        time.sleep(0.5)  # give it some time to actually change the value
```

Comment:
That shouldn't be necessary. The setter is blocking. The VA value will be updated once the setter returns.